brew/install-start-with-checks.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+set -eu
+
+eval $(brew/vars.sh)
+
+echo
+echo brew version...
+brew --version
+
+set +e # to turn -e off (relax this only for time being)
+brew doctor # TODO(mkramlich): for brew maintainer, best if have no findings and exit 0
```

Comment:
Then why turn off -e in the first place? If homebrew is not in a cromulent state why should we continue to build?